src/HexagonOffload.cpp

```diff
@@ -356,7 +428,8 @@ Stmt inject_hexagon_rpc(Stmt s, const Target &host_target) {
         Target::NoAsserts,
         Target::HVX_64,
         Target::HVX_128,
-        Target::HVX_v62
+        Target::HVX_v62,
```

Comment:
I think appearances of HVX_v62 are OK as it is now public.  Fully supporting HVX_v62 is probably worth putting into a separate patch however.   Otherwise, LGTM